llama_index/vector_stores/qdrant.py

```diff
@@ -432,6 +560,34 @@ def parse_to_query_result(self, response: List[Any]) -> VectorStoreQueryResult:
 
         return VectorStoreQueryResult(nodes=nodes, similarities=similarities, ids=ids)
 
+    def deduplicate_hybrid_result(
```

Comment:
if we do dedup or fusion
* top k or sparse top k will not represent how many we could return 
* similarities is mixed with sparse distance and dense distance
do we need to separate the nodes returned from each side. or should we give them a label about which retrieval source this node comes from

Reply:
Our other hybrid approaches work the same way (returning a single list). Not sure I want to introduce some extra concept of labelling quite yet.

Ideally, the user provides some function that handles the normalizing of scores/ordering. But I should probably figure out a better default than deduplicate